src/bindings/js/node/tests/basic.test.js

```diff
@@ -186,3 +186,17 @@ describe('Input class for ov::Input<const ov::Node>', () => {
   });
 
 });
+
+it('Test exportModel()/importModel()', () => {
+  const userStream = compiledModel.exportModel();
+  const newCompiled = core.importModel(userStream, 'CPU');
```

Comment:
Because these methods consume some time to execute, should they be async?

Reply:
I will rename those to `exportModelSync(`) and `importModelSync()`